spec/anchor-preview.spec.js

```diff
@@ -40,7 +40,7 @@ describe('Anchor Preview TestCase', function () {
             expect(anchorPreview.showPreview).toHaveBeenCalled();
 
             // link is set in preview
-            expect(anchorPreview.getPreviewElement().querySelector('i').innerHTML).toBe(document.getElementById('test-link').attributes.href.value);
+            expect(anchorPreview.getPreviewElement().querySelector(anchorPreview.previewValueSelector).innerHTML).toBe(document.getElementById('test-link').attributes.href.value);
```

Comment:
I'm wondering if we should still be using `a` instead of referencing what's inside the anchorPreview itself.  I'm assuming the tests broke when you changed away from a `<i>` and perhaps that's what we would want should someone attempt to change the DOM again?

What do you think?